module-system/sov-modules-core/src/storage/scratchpad.rs

```diff
@@ -338,6 +362,28 @@ impl<C: Context> WorkingSet<C> {
         AccessoryWorkingSet { ws: self }
     }
 
+    /// Returns a handler for the archival state (JMT state).
+    pub fn archival_state(&mut self, version: Version) -> ArchivalJmtWorkingSet<C> {
```

Comment:
it looks like `archival_accessory_state & archival_state` methods cane be private.